build-scripts/copyNonCodeFiles.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/*
+    This script is called from npm run compile.
+    It copies non-code files to out/.
```

Comment:
Maybe specify that this only copies files in the `relativePaths` array, right now it sounds like it should copy everything.

Reply:
✔️